### PR TITLE
Add option to hide pages from view/export

### LINKED ISSUE #648: [composer] Add option to hide pages from view/export
https://github.com/qgis/QGIS/commit/2215d487124b0800b1ea8420a16f8ed05085128c
Author: Nyall Dawson
Date:   Mon Sep 7 07:55:40 2015 +1000
    
    Sponsored by NIWA

### PR DESCRIPTION
**Pull request by yjacolin**
* Add all missing items in view menu
* Update the mActionZoomToSelected.png icon

Fix #648

### CODE REVIEW

**Comment by DelazJ** [comment]
Not the same icons.
I think [mActionZoomToSelected ](https://github.com/qgis/QGIS/blob/master/images/themes/default/mActionZoomToArea.svg) has been replaced in composer by [mActionZoomToArea](https://github.com/qgis/QGIS/blob/master/images/themes/default/mActionZoomToArea.svg)

File: source/docs/user_manual/print_composer/overview_composer.rst [hunk 1]
```diff
@@ -327,8 +327,25 @@ To navigate in the canvas layout, the Print Composer provides some general tools
 * |zoomFullExtent| :sup:`Zoom Full`
 * |zoomActual| :sup:`Zoom to 100%`
 * |draw| :sup:`Refresh view` (if you find the view in an inconsistent state)
-* |pan| :sup:`Pan composer`
-* |zoomToSelected| :sup:`Zoom` (zoom to a specific region of the Composer)
+* `Show Grid` behind items.
+* `Snap Grid` to snap items on the grid.
+* `Show Guides` to help user to align items. Theses are red line that you can
```
Comment (DelazJ) [inline source/docs/user_manual/print_composer/overview_composer.rst]: typo: These

File: source/docs/user_manual/print_composer/overview_composer.rst [hunk 2]
```diff
@@ -327,8 +327,25 @@ To navigate in the canvas layout, the Print Composer provides some general tools
...
+  create click in the rule (above or at the left side of the layout) and drag
```
Comment (DelazJ) [inline source/docs/user_manual/print_composer/overview_composer.rst]: ~~create~~

**Comment by yjacolin** [comment]
@DelazJ I also updated the icon file, so the icon is not correct now?

**Comment by DelazJ** [comment]
I meant you should create a new substitution zoomToArea. The mActionZoomToSelected is no longer the icon used in composer so all zoomToSelected occurrences should also be replaced in print composer chapters

**Comment by yjacolin** [comment]
Do I need to remove mActionZoomToSelected from conf.py?

**Comment by DelazJ** [comment]
no;it's used in the map canvas, attribute table... we have now two icons, mActionZoomToSelected (which was previously used also for print composer) and mActionZoomToArea dedicated to only composers

**Comment by yjacolin** [comment]
@DelazJ done, I successfully revert my change for the icon :) I was a little bit afraid of this.

File: source/conf.py [hunk 3]
```diff
@@ -1103,7 +1103,8 @@
    :width: 1.5em
 .. |zoomToSelected| image:: /static/common/mActionZoomToSelected.png
    :width: 1.5em
-
+.. |zoomToArea| image:: /static/common/mActionZoomToArea.png
+   :width: 1.5em
```
Comment (DelazJ) [inline source/conf.py]: alphabetic sorting? :)

File: source/docs/user_manual/print_composer/overview_composer.rst [hunk 4]
```diff
@@ -327,8 +327,25 @@ To navigate in the canvas layout, the Print Composer provides some general tools
...
 * |zoomActual| :sup:`Zoom to 100%`
 * |draw| :sup:`Refresh view` (if you find the view in an inconsistent state)
-* |pan| :sup:`Pan composer`
-* |zoomToSelected| :sup:`Zoom` (zoom to a specific region of the Composer)
+* `Show Grid` behind items.
+* `Snap Grid` to snap items on the grid.
+* `Show Guides` to help user to align items. These are red line that you can
+  click in the rule (above or at the left side of the layout) and drag and drop
+  to the desired location.
+* `Snap Guides` allow user to snap items on guides.
```
Comment (DelazJ) [inline source/docs/user_manual/print_composer/overview_composer.rst]: allow**s**?

File: source/docs/user_manual/print_composer/overview_composer.rst [hunk 5]
```diff
@@ -327,8 +327,25 @@ To navigate in the canvas layout, the Print Composer provides some general tools
...
+* `Clear Guides` to remove all current guides.
+* `Show Bounding box` around the items.
+* `Show Rules` around the layout.
+* `Show Pages` or set up pages to transparent. Often composer is used
+  to create non-print layouts, e.g. for inclusion in presentations or other
+  documents, and it's desirable to export the composition using a totally
+  transparent background. It's sometimes referred to as "infinite canvas" in
+  other editing packages.
+* `Toggle Full Screen` make the composer window to full screen.
+* `Hide Panels` hide/shows the right panel
```
Comment (DelazJ) [inline source/docs/user_manual/print_composer/overview_composer.rst]: hides/shows all the panels

**Comment by yjacolin** [comment]
Ready to be merged after test passed.

**Pull request merged**